conf/modules.config

```
@@ -259,7 +259,16 @@ process {
         ]
     }
 
-    withName: "SAMTOOLS_INDEX_MEM" {
+    withName: BOWTIE2_ALIGN {
+        ext.args = {[ "-N ${params.mapping_bowtie2_n} -L ${params.mapping_bowtie2_l} --trim5 ${params.mapping_bowtie2_trim5} --trim3 ${params.mapping_bowtie2_trim3} --maxins ${params.mapping_bowtie2_maxins}",
```

We are missing the readgroup definitions in the `args` too

---

```suggestion
        ext.args = {[ "-N ${params.mapping_bowtie2_n} -L ${params.mapping_bowtie2_l} --trim5 ${params.mapping_bowtie2_trim5} --trim3 ${params.mapping_bowtie2_trim3} --maxins ${params.mapping_bowtie2_maxins}" --rg-id '@RG\\tID:ILLUMINA-${meta.library_id}\\tSM:${meta.sample_id}\\tPL:illumina\\tPU:ILLUMINA-${meta.library_id}-${meta.strandedness}' ,
```
this is fixed, right? aka no user option